packages/app/src/hooks/useSyncCurrentStateIntoOpenedProjects.ts

```diff
@@ -1,7 +1,8 @@
+import { type Project } from '@ironclad/rivet-core';
 import { useEffect, useState } from 'react';
 import { useRecoilState, useRecoilValue } from 'recoil';
 import {
-  loadedProjectState,
+  loadedProjectState, type OpenedProjectInfo,
```

Comment:
prettier?

Reply:
fixed.

I would like to get this issue done, and make lint/pretty part of the PR review process. I've seen people setup PRs so that comments for lint problems are auto added. That way we can automate these sorts of things.

Also, the entire code base could use a pretty pass because if I run it on the code now it generates a lot of changed files. We need to do a PR to clean it up and get it merged so we can stabilize the main branch.

Reply:
Ah, yeah, it's a prettier version difference problem. My editor is configured to run for Ironclad's prettier version which is a little behind. I'd love to have CI do that stuff too @codemile